source/includes/DataIndexing/_pushurls.md

```diff
@@ -26,4 +26,12 @@ Crawler ID | The ID of the specific crawler you'd like to crawl the given URLs
 
 #### HTTP body
 
-The body content is simply a newline list of urls to crawl.
\ No newline at end of file
+The pushurls takes a collection of pages, if it’s one page
+
+> The request body would look like this:
+
+```json
+[
+  "https://example.com/page/to/index"
+]
+```
```

Comment:
The suggested phrasing on the bug was a bit awkward/confusing -- it makes it seem like the pushurls takes a collection _only if_ it's a single page. Could you change the punctuation to:

Pushurls takes a collection of pages.
> If it’s one page, the request body would look like this:
```json
[
  "https://example.com/page/to/index"
]
```